### ISSUE TITLE
unable to remove duplicates

### ISSUE THREAD

**Issue by 150d** [stevencohn/OneMore#1604]
Hi,

I'm unable to use the "find duplicate pages" function in OneMore:

- The first dialog comes up, allowing to set the compare mode and what to compare (notebooks or sections)
- Scanning starts, in my case taking quite some time (I have a few large sections that are close to 100% identical)
- ... and then nothing. Scanning finishes, the progress bar vanishes, no results are shown. Additionally, OneMore is unresponsive: I can open its (main) menu, but nothing works, even "About" doesn't open its info dialog any more.

What should i do next, how can I see the results of the scan?

**Comment by stevencohn** [comment]
Can you attach the `%TEMP%\OneMore.log` log file here?

**Comment by 150d** [comment]
Of course:
[OneMore.log](https://github.com/user-attachments/files/17269710/OneMore.log)

**Comment by stevencohn** [comment]
<img width="525" alt="image" src="https://github.com/user-attachments/assets/c3a85944-60c4-4f83-98ee-9a219f023766">

**Comment by 150d** [comment]
Indeed, shouldn't there be a "results" dialog coming up after completion?

**Comment by stevencohn** [comment]
I've tried to repro this but perhaps don't have a large enough data set. I only have a mere 1800 pages across 10 notebooks. Without further information or the ability to repro, I'm kind of stuck. If you find more info, please open a new issue.
